includes/wc-update-functions.php

```diff
@@ -1932,3 +1932,17 @@ function wc_update_354_modify_shop_manager_caps() {
 function wc_update_354_db_version() {
 	WC_Install::update_db_version( '3.5.4' );
 }
+
+/**
+ * Update product lookup tables in bulk.
+ */
+function wc_update_360_product_lookup_tables() {
+	wc_update_product_lookup_tables();
```

Comment:
Not seeing any use of this function elsewhere so assume this is just for the initial population? If that is the case probably makes sense to just the code inside here instead of defining a function in the product functions file?

Reply:
Hmm actually I think it's a good idea to have that function handy if for some reason a plugin wants to update the lookup table as well

Reply:
That was my thinking @kloon @Tofandel. It could be called manually if say we added a tool to regenerate the table data.